Jason/lab1/Part2/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.10)
+project(Lab1_Hard)
+
+# Define the library
+add_library(hello_world STATIC src/hello_world.cpp)
+
+# Specify include directories for this library
+target_include_directories(hello_world PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/includes)
```

Comment:
no need to use the cmake variable pointing out the current source directory. also by convention the include folder should be named `include`